### PR TITLE
spc2pdf implementation

### LINKED ISSUE #439: SPC inspector producing graphs from .spc files

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #439

### CODE REVIEW

File: pytrip/utils/spc2pdf.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+#
+#    Copyright (C) 2010-2017 PyTRiP98 Developers.
```
Comment (nbassler) [inline pytrip/utils/spc2pdf.py]: 2018

File: pytrip/utils/spc2pdf.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    with PdfPages(parsed_args.pdf_file.name) as pdf:
+        logging.debug("Saving PDF file {}".format(parsed_args.pdf_file.name))
+        plt.rc('text', usetex=False)
+
+        # first a summary plot of fluence vs depth, log scale on fluence (Y), several series corresponding to Z species
+        fig, ax = plt.subplots()
+        ax.set_title("")
+        ax.set_xlabel("Depth [cm]")
+        ax.set_ylabel("Fluence [a.u.]")
+        ax.set_yscale('log')
```
Comment (grzanka) [inline pytrip/utils/spc2pdf.py]: I will also consider plotting fluence on linear scale by default and give the user a chance to change to scale to the logarithmic. @nbassler what do you think ?

File: pytrip/utils/spc2pdf.py [hunk 3]
```diff
@@ -26,20 +26,29 @@
...
+    import matplotlib.pyplot as plt
+    from matplotlib import colors
+    from matplotlib.backends.backend_pdf import PdfPages
+
     # parse arguments
     parser = argparse.ArgumentParser()
     parser.add_argument("spc_file", help="location of input SPC file", type=argparse.FileType('r'))
     parser.add_argument("pdf_file", help="location of output PDF file", type=argparse.FileType('w'))
+    parser.add_argument('-l', '--logscale', help="Enable fluence plotting on logarithmic scale", action='store_true')
+    parser.add_argument('-c', '--colormap', help='image color map, see http://matplotlib.org/users/colormaps.html '
```
Comment (nbassler) [inline pytrip/utils/spc2pdf.py]: nice

**Comment by grzanka** [comment]
It can be also added as an option. PDF has this nice feature that it has many pages. Single SPC - single PDF with many plots.

**Pull request merged**